### PR TITLE
Create the Login functionality, made the Router Wrapp for the App, Ge…

### PR DESCRIPTION
**Pull request by AlexGioffDev**
Create the Login functionality, made the Router for the App, Get info from the account after the authorization

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -6,16 +6,34 @@ import {
   ButtonLogin,
   SpotifyButton,
   Backward} from "../../styled/Utils";
-
+import querystring from 'querystring';
 
 const Login = () => {
+  
+  const goToLogin = () => {
+    let state = 'some-state-of-my-choice';
```
Comment (themythia) [inline src/components/Login/Login.js:15]: This should be a random string that's not visible (like clientId) on release version.

https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.1

File: src/components/Authorize/Authorize.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    request.get(options, function(error, response, body) {
+      console.log(body);
+    });
+  }, []);
+
+  return (
+    <div>
+      <h1>Hello</h1>
+    </div>
+  )
```
Comment (themythia) [inline src/components/Authorize/Authorize.js]: I think this should redirect to the `HomePage` when auth succeeds. Also we need error handling if auth fails.

File: src/styled/Modal.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  height: 50%;
+  background: white;
+  border: 1px solid black;
+  padding: 7.5px 20px;
+  position: absolute;
+  border-radius: 14px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`;
```
Comment (themythia) [inline src/styled/Modal.js:29]: I believe this is unused?

**Review by themythia — COMMENTED** [review]
Overall looks good, good job Alex.

File: src/styled/Modal.js (hunk 3 above)
Comment (AlexGioffDev) [reply src/styled/Modal.js:29]: yes i forget to delete this after choice to not continue without the modal

File: src/components/Authorize/Authorize.js (hunk 2 above)
Comment (AlexGioffDev) [reply src/components/Authorize/Authorize.js]: yes, i do this page for testing so on the console log you can see if everything it's working, with react context we can set the state of the authenitcation so we can redirect on the HomePage after login success

File: src/components/Login/Login.js (hunk 1 above)
Comment (AlexGioffDev) [reply src/components/Login/Login.js:15]: yes, for testing we use things like this for testing and development, then with react context we set the state and use it for checking

**Pull request closed without merging**